### PR TITLE
Update class paths

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/174/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: test_cromwell.sh [hunk 1]
```diff
@@ -93,7 +93,7 @@ if [[ -n ${CROMWELL_BRANCH} ]]; then
     echo "Building Cromwell"
     sbt clean assembly >> "${ASSEMBLY_LOG}" 2>&1
     cd ..
-    CROMWELL_JAR=$(find "${RUN_DIR}"/cromwell/target/scala-2.11 -name "cromwell-*.jar")
+    CROMWELL_JAR=$(find "${RUN_DIR}"/cromwell/target/scala-2.* -name "cromwell-*.jar")
```
Comment (mcovarr) [inline test_cromwell.sh:96]: why is this one a glob?

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/174/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: test_cromwell.sh (hunk 1 above)
Comment (ruchim) [reply test_cromwell.sh:96]: Discussed in person, but to avoid having to update this in the future when Cromwell is updated from 2.11 to 2.x

**Pull request merged**